client/internal/ingressgw/manager_test.go

```diff
@@ -0,0 +1,244 @@
+package ingressgw
+
+import (
+	"fmt"
+	"net/netip"
+	"testing"
+
+	firewall "github.com/netbirdio/netbird/client/firewall/manager"
+)
+
+var (
+	_ firewall.Rule = &MocFwRule{}
```

Comment:
```suggestion
	_ firewall.Rule = (*MocFwRule)(nil)
```
might be a bit better